### DISCUSSION TITLE
Premium PDF use Brave instead of Chrome?

### DISCUSSION THREAD

**Discussion by BKLTMS** [YetiForceCompany/YetiForceCRM#17161] [General]
Can a Chrome based browser like Brave be used instead of the actual Chrome browser for Premium PDF?
If yes, what settings/entries would be used in place of the chrome entries in php files.?
If no, can only Chrome be used and no other fork of Chrome?

Thanks in advance!

**Comment by rskrzypczak** [comment]
> Can a Chrome based browser like Brave be used instead of the actual Chrome browser for Premium PDF?

No.

> If no, can only Chrome be used and no other fork of Chrome?

chrome/chromium 
https://doc.yetiforce.com/administrator-guides/system-tools/pdf/#configuring-the-pdf-generator-path

**Comment by BKLTMS** [comment]
Is Chromium okay to use instead of Chrome, as Chromium is less intrusive?
Thank you in advance!

**Reply by rskrzypczak** [reply]
Yes.
This is described in the mentioned documentation.
Go through it again, only this time more carefully :)

**Comment by BKLTMS** [comment]
Thank you for your response.

The docs were reviewed, before the question, and Chromium is mentioned within them.
 However, there are differences between Chrome and Chromium.  I guess a better question would be 'is there any loss in functionality or drawback to using Chromium instead of Chrome'?
Is it better to use one over the other?

While there is documentation to an extent, for some of us it does not always paint a clear picture of what the best choice may be or clear up a question that may arise.  If we ask a question it is because we have reviewed the documentation and performed in depth searches to find an answer or direction before we proceed.  If that proves to not provide an answer or direction, we then ask the question to those that can hopefully be helpful. 

Best regards.

**Comment by BKLTMS** [comment]
I am hoping this question is okay within the continuation of this thread....

I have installed (on local C: drive) the latest stable version of 'ungoogled-chromium' and has been confirmed as functioning okay.  YetiForce is installed on a hosted VPS and has/is functioning fine.  I have configured the  'pdf.php' file for binary location. Have tried several different changes to the file, however, nothing works to allow the 'Generating engine' selection to show the Chromium choice. Only Yetiforce PDF continues to be offered.

This must be simple, please advise?

Thanking you in advance for your help!
Best regards.

**Comment by rskrzypczak** [comment]
YetiForce must be registered in accordance with the license requirements.
This is one of the requirements for the availability of this functionality.
Since you set the configurations for chromium in YetiForce, the lack of registration is probably the reason.
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/ddd93b69-4e15-4a70-abac-d3e1709ba62d)

**Comment by BKLTMS** [comment]
It is and has been registered since installation and status shows as 'Accepted'. Another add-on we purchased is working, so I do not believe it is a registration issue.  If it is, it  has to be on YetForce's end, as I confirmed 'registration' when ordering the "Addons".

I followed the instructions for Premium PDF setup and searched extensively for information on what to do.  The choice for the 'Chromium' engine is not being offered.  Suggestions please.

Thank you and appreciated.
